README.md

```diff
@@ -91,18 +91,32 @@ After that you have the following instance methods available to you.
     
     `TelephoneNumber.parse("3175082237", :US)`
     
+    If you pass an E164 formatted number, we will determine the country on the fly. 
+    
+    `TelephoneNumber.parse("+13175082237")`
+    
   - #### `valid?`
     
     Returns boolean value indicating whether or not a particular number is valid.
     
     `TelephoneNumber.valid?("3175082237", :US) ==> true`
     
+    If you are looking to validate against a specific set of keys, you can pass in an array of keys
+    
+    `TelephoneNumber.valid?("3175082237", :US, [:mobile, :fixed_line]) ==> true`
```

Comment:
gahhh thanks for catching that. I could've sworn it didn't render like this when I edited it.